readthedocs/restapi/views/footer_views.py

```diff
@@ -43,7 +43,7 @@ def get_version_compare_data(project, base_version=None):
     }
     if highest_version_obj:
         ret_val['url'] = highest_version_obj.get_absolute_url()
-        ret_val['slug'] = highest_version_obj.slug,
+        ret_val['slug'] = highest_version_obj.slug
```

Comment:
This looks like an unrelated bug?

Reply:
Ha! Yeah. I don't understand how this passed the test in the PR that was merged :/